news/2020-06-26-mappers-report-May-June.md

```diff
@@ -0,0 +1,201 @@
+- **osu!taiko:** [realy0\_](https://osu.ppy.sh/users/8863359), [\_DUSK\_](https://osu.ppy.sh/users/6092181), [davidminh0111](https://osu.ppy.sh/users/9623142), [Cychloryn](https://osu.ppy.sh/users/6921736) & [Axer](https://osu.ppy.sh/users/7299864)
+- **osu!catch:** [Baron](https://osu.ppy.sh/users/10286499), [Secre](https://osu.ppy.sh/users/2306637) & [Bastian](https://osu.ppy.sh/users/6345176)
+- **osu!mania:** [Kyousukee](https://osu.ppy.sh/users/8842107), [Shima Rin](https://osu.ppy.sh/users/6089608), [Garalulu](https://osu.ppy.sh/users/757783) & [Muse Dash](https://osu.ppy.sh/users/13695676)
+
+Do you want to see your own name on that list too? And do you like modding and helping other people with their beatmaps? Then become a Beatmap Nominator! The applications are always open. Head over to the [Beatmap Nominator website](https://bn.mappersguild.com/) and apply today!
+
+### BN Resignations
+
+- **osu!:** [Affirmation](https://osu.ppy.sh/users/6186628)
+- **osu!taiko:** [Cynplytholowazy](https://osu.ppy.sh/users/3901754) & [Gamelan4](https://osu.ppy.sh/users/9856910)
+- **osu!catch:** [-Luminate](https://osu.ppy.sh/users/4778689)
+- **osu!mania:** [\_Kobii](https://osu.ppy.sh/users/6209713) & [Unpredictable](https://osu.ppy.sh/users/7560872)
+
+### BN and QAT/NAT Badges
+
+Recently a new wave of badges acknowledging the length of time members have put into teams for managing ranked! Those who are on this list have been contributing to the community in so many ways during their time on osu!, and fully deserve this acknowledgement. We thank them all for their contribution so far and hope to see much more of them in the future! Here is the list of those:
+
+- **BN 1 Year Badge:** [incandescence](https://osu.ppy.sh/users/6256027), [Kaitjuh](https://osu.ppy.sh/users/2225327), [Unpredictable](https://osu.ppy.sh/users/7560872), [Nuvolina](https://osu.ppy.sh/users/10974170) & [Dored](https://osu.ppy.sh/users/10284894)
+- **BN 2 Year Badge:** [JBHyperion](https://osu.ppy.sh/users/4879508), [Nevo](https://osu.ppy.sh/users/7451883), [Mordred](https://osu.ppy.sh/users/7265097), [bossandy](https://osu.ppy.sh/users/360437) & [Namki](https://osu.ppy.sh/users/5248582)
+- **BN 3 Year Badge:** [Benita](https://osu.ppy.sh/users/4023183) & [Sonnyc](https://osu.ppy.sh/users/11771)
+- **BN 4 Year Badge:**  [Nozhomi](https://osu.ppy.sh/users/2716981)
+- **QAT/NAT 1 Year Badge:** [Ascendance](https://osu.ppy.sh/users/2931883), [Noffy](https://osu.ppy.sh/users/1541323) & [Faputa](https://osu.ppy.sh/users/845733)
+- **QAT/NAT 2 Year Badge:** [Lasse](https://osu.ppy.sh/users/896613), [-Mo-](https://osu.ppy.sh/users/2202163) & [Chaoslitz](https://osu.ppy.sh/users/3621552)
+- **QAT 3 Year Badge:** [OnosakiHito](https://osu.ppy.sh/users/290128)
+
+## <a id="mappers-showcase"></a>Mappers' Showcase
+
+The Mappers' Showcase parades lesser-known mappers from every game mode. Since we bundled two issues in one post, this month will feature two game modes. 
```

Comment:
```suggestion
The Mappers' Showcase presents lesser-known mappers from every game mode. Since we bundled two issues in one post, his month we will feature two game modes. 
```